docs/snippets/all/archetypes/geo_line_string_simple.py

```diff
@@ -0,0 +1,20 @@
+"""Log a simple geospatial line string."""
+
+import rerun as rr
+
+rr.init("rerun_example_geo_line_strings", spawn=True)
+
+rr.log(
+    "colorado",
+    rr.GeoLineStrings(
+        [
```

Comment:
```suggestion
    rr.GeoLineStrings(
        lat_long=[
```

Reply:
For python, I considered many alternatives, non of which I liked. I finally opted to add an explicit `from_lat_lon()` constructor, throwing in a `from_lon_lat()` one for good measure. The `__init__` is unchanged.

Reply:
Why not just have all arguments named with a `**`?

Reply:
Because that would imply, in the argument name, replacing its nature by its type (`positions` -> `lat_lon` / `line_stings` -> `lat_lon`), and I'm not sure it's a win. I guess one approach is to use `positions_lat_lon`/`line_strings_lat_lon`, but this is a bit ugly imo.